Library/Homebrew/utils.rb

```diff
@@ -517,3 +517,18 @@ def migrate_legacy_keg_symlinks_if_necessary
   end
   FileUtils.rm_rf legacy_pinned_kegs
 end
+
+def puts_hash(hash, indent: 0)
+  return hash unless hash.is_a? Hash
+  hash.each do |key, value|
+    printf "#{" " * indent*2}#{key}:"
```

Comment:
Might want to use some brackets here just to be totally clear what's going on with the operator overloads.